tests/stress_tests/scripts/memcheck_upload.py

```diff
@@ -203,7 +201,8 @@ def main():
     parser.add_argument('--db_url', required=not is_dryrun,
                         help='MongoDB URL in a for "mongodb://server:port".')
     parser.add_argument('--db_collection', required=not is_dryrun,
-                        help=f'Collection name in {DATABASE} database to upload')
+                        help=f'Collection name in {DATABASE} database to upload.'
+                             f' Available values: "commit", "nightly", "weekly"')
```

Comment:
Better to use `choices`: https://docs.python.org/3/library/argparse.html#choices

In that case Argparse will automatically verify and form proper help message

Reply:
Agree. Done